src/middleware/hasAuth.js

```diff
@@ -0,0 +1,14 @@
+const Response = require("../utils/responseHandler");
+
+/* THIS MIDDLEWARE IS TO VERIFY IF USER IS LOGGED IN OR NOT */
+exports.hasAuth = function(req, res, next) {
+    if (req.user) {
+        next();
+    } else {
+        return Response.send(
+            res,
+            200,
```

Comment:
Status code should be 401 (not authorized) or 403 (forbidden)

Reply:
Fixed